playbooks/ansible-test-base/run.yaml

```diff
@@ -1,10 +1,21 @@
 ---
 - hosts: controller
   tasks:
-    - name: Fetch and install the artifacts
-      import_role:
-        name: deploy-artifacts
-      when: ansible_test_collections | default(False)
+    - when: ansible_test_collections | default(False)
+      block:
+        - name: Fetch and install the artifacts
+          import_role:
+            name: deploy-artifacts
+        - name: Copy the galaxy.yml on the controller
```

Comment:
Can't this be simplified using slurp?

Reply:
Not really, `slurp` stores in memory, `include_vars` reads from a file.